src/main/java/uk/gov/hmcts/reform/iacaseapi/infrastructure/utils/GetStaffLocation.java

```
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.reform.iacaseapi.infrastructure.utils;
+
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.HearingCentre;
+
+public class GetStaffLocation {
+
+    private GetStaffLocation() {
+
+    }
+
+    public static String getStaffLocation(HearingCentre hearingCentre) {
+        String centre = hearingCentre.getValue();
+        switch (centre) {
+            case "birmingham":
+            case "nottingham":
+            case "coventry":
+                return "Birmingham";
+            case "glasgow":
+            case "glasgowTribunalsCentre":
+            case "belfast":
+                return "Glasgow";
+            case "bradford":
+                return "Bradford";
+            case "hattonCross":
+                return "Hatton Cross";
+            case "manchester":
+                return "Manchester";
+            case "newcastle":
+                return "Newcastle";
+            case "newport":
+                return "Newport";
+            case "taylorHouse":
+                return "Taylor House";
+            default:
+                return null;
```

Maybe this should throw an exception